pvlib/test/test_atmosphere.py

```diff
@@ -113,3 +111,82 @@ def test_first_solar_spectral_correction_supplied():
 def test_first_solar_spectral_correction_ambiguous():
     with pytest.raises(TypeError):
         atmosphere.first_solar_spectral_correction(1, 1)
+
+
+def test_kasten96_lt():
+    """Test Linke turbidity factor calculated from AOD, Pwat and AM"""
+    timestamps = pd.DatetimeIndex(MELBOURNE_FL[0])
+    timestamps = timestamps.tz_localize('UTC').tz_convert('Etc/GMT+5')
+    melbourne_fl = pd.DataFrame(MELBOURNE_FL[1], index=timestamps)
+    aod_bb = melbourne_fl['AOD']
+    pwat_cm = melbourne_fl['Pwat']
+    pressure = melbourne_fl['Pressure']
+    dry_temp = melbourne_fl['DryBulb']
+    lat, lon, alt = 28.117, -80.65, 11.0
+    sp = solarposition.get_solarposition(
+        timestamps, lat, lon, alt, pressure=pressure, temperature=dry_temp
+    )
+    am = atmosphere.relativeairmass(sp.apparent_zenith)
+    amp = atmosphere.absoluteairmass(am, pressure=pressure)
+    lt_kasten = atmosphere.kasten96_lt(
+        airmass_absolute=amp, precipitable_water=pwat_cm, aod_bb=aod_bb
+    )
+    lt = clearsky.lookup_linke_turbidity(timestamps, lat, lon)
+    assert np.allclose(lt, lt_kasten, 1e-3)
+    return lt, lt_kasten
+
+
+def test_angstrom_aod():
+    """Test Angstrom turbidity model functions."""
+    aod550 = 0.15
+    aod1240 = 0.05
+    alpha = atmosphere.angstrom_alpha(aod550, 550, aod1240, 1240)
+    np.isclose(alpha, [1.3513924317859232])
```

Comment:
It's that you have to assert something is true or not. np.isclose only returns an array.